outdoorsmanship.lic

```diff
@@ -9,21 +9,44 @@ class Outdoorsmanship
   include DRCI
 
   def initialize
+    arg_definitions = [
+      [
+        { name: 'mindstates', regex: /\d+/, optional: true, description: 'Number of mindstates or collection attempts to perform before exiting. Defaults to 3' },
+        { name: 'collect_item', regex: /\w+/i, optional: true, description: 'Item to collect. Will use forage_item if not set' }
+      ]
+    ]
+    args = parse_args(arg_definitions)
+
     settings = get_settings
-    @forage_item = settings.forage_item
+    
+    @targetxp = 3
```

Comment:
💡 These two lines could be combined, e.g. `@targetxp = args.mindstates ? args.mindstates.to_i : 3`

Reply:
I typically avoid the ternary operator because I think it reduces readability, although what I have in there isn't much better.  Up to you if you want me to correct it.

Reply:
I find 'simple' (and I consider this to be 'simple enough') ternaries easier to read, but it's definitely a preference thing.

💡 I wonder if coalesce would work here? `@targetxp = args.mindstates || 3`